src/core/core-serialize.c

```diff
@@ -17,6 +17,17 @@
 #include <zlib.h>
 #endif
 
+#ifdef __CELLOS_LV2__
+#include <sys/sys_time.h>
+int gettimeofday(struct timeval* tv, void* unused) {
```

Comment:
I would recommend migrating this to `formatting.c` as well, just so that it's in a consistent place with the other time function.

Reply:
Done.